src/js/components/awardv2/idv/activity/chart/ActivityChart.jsx

```diff
@@ -177,6 +177,7 @@ export default class ActivityChart extends React.Component {
             const start = xScale(bar._startDate.valueOf()) + padding.left;
             const end = xScale(bar._endDate.valueOf()) + padding.left;
             data.barWidth = end - start;
+            if (data.barWidth < 1.5) data.barWidth = 1.5;
```

Comment:
In Staging, the bars appear at 1.44px; perhaps suggest making them a bit bigger.

As far as the code is concerned, this appears pretty trivial so I'm down to just agree w/ Marco but IMO this is still too small.